scripts/perftune.py

```diff
@@ -575,11 +581,11 @@ def __setup_rfs(self, iface):
         ethtool_msg = "Enable ntuple filtering HW offload for {}...".format(iface)
         if dry_run_mode:
                 perftune_print(ethtool_msg)
-                run_one_command(['ethtool','-K', iface, 'ntuple', 'on'], stderr=subprocess.DEVNULL)
+                run_ethtool(['-K', iface, 'ntuple', 'on'])
```

Comment:
You didn't shut up the `stderr` as it was before.

Reply:
In addition to that this patch changes the behaviour of the `dry run mode` - run_read_only_command() always executes a command while run_command only prints the command in a dry run mode.

Reply:
addressed in the same commit
squashed with
"don't make changing ethtool as read only"